src/plugins/index.js

```diff
@@ -7,38 +7,39 @@ import interactive from './interactive-plugin';
 import './videojs-http-source-selector/plugin';
 // #endif
 
+import aiHighlightsGraph from './ai-highlights-graph';
+import analytics from './analytics';
 import autoplayOnScroll from './autoplay-on-scroll';
-import floatingPlayer from './floating-player';
-import colors from './colors';
-import contextMenu from './context-menu';
+import chapters from './chapters';
 import cloudinary from './cloudinary';
-import analytics from './analytics';
 import cloudinaryAnalytics from './cloudinary-analytics';
-import vttThumbnails from './vtt-thumbnails';
-import aiHighlightsGraph from './ai-highlights-graph';
-import chapters from './chapters';
+import colors from './colors';
+import contextMenu from './context-menu';
+import floatingPlayer from './floating-player';
 import pacedTranscript from './paced-transcript';
+import vttThumbnails from './vtt-thumbnails';
 
+// Lazy loaded plugins
 import styledTextTracks from './styled-text-tracks';
 
 const plugins = {
+  // #if (!process.env.WEBPACK_BUILD_LIGHT)
```

Comment:
not sure i understand why do we have these changes?

Reply:
You're right, apologies :)
Some commits here were made just to trigger a build, I could have cleaned it up but they were nice cleanups, here for example I sorted the plugins alphabetically